README.md

```diff
@@ -7,8 +7,7 @@ The `ec2-spot-interrupter` is a simple CLI tool that triggers Amazon EC2 Spot In
 ## Installation
 
 ```bash
-brew tap aws/tap
-brew install aws/ec2-spot-interrupter
+brew install tap/aws/ec2-spot-interrupter
```

Comment:
Need to replace `tap/aws` with `aws/tap` (same as the `brew tap` command being removed):

```suggestion
brew install aws/tap/ec2-spot-interrupter
```

I tested both ways to verify this syntax:
```
$ brew install tap/aws/ec2-spot-interrupter
==> Tapping tap/aws
Cloning into '/usr/local/Homebrew/Library/Taps/tap/homebrew-aws'...
remote: Repository not found.
fatal: repository 'https://github.com/tap/homebrew-aws/' not found
Error: Failure while executing; `git clone https://github.com/tap/homebrew-aws /usr/local/Homebrew/Library/Taps/tap/homebrew-aws --origin=origin --template=` exited with 128.
$ brew install aws/tap/ec2-spot-interrupter
Warning: aws/tap/ec2-spot-interrupter 0.0.8 is already installed and up-to-date.
To reinstall 0.0.8, run:
  brew reinstall ec2-spot-interrupter
```

Reply:
I just removed the prefix since it can be a local reference now. Let me if that is sufficient